### PR TITLE
Add documentation about client-side deploy

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: source/clientsideapp.md [hunk 1]
```diff
@@ -105,7 +105,69 @@ Also, an `index.html` file needs to be at the root of your repository with your
...
+
+```
+cd myapp
+cozy-dev deploy ./build
+```
+
+
+#### 3. Distribute it _via_ Github
+
+This last solution allow you to install you app in any Cozy, but any changes need you to push you new version to Github before updating it in your Cozy. It can quickly become a painful back andforth, be careful if you choose it.
```
Comment (nono) [inline source/clientsideapp.md]: s/andforth/and forth/

**Review by nono — APPROVED** [review]

File: source/clientsideapp.md [hunk 2]
```diff
@@ -105,7 +105,69 @@ Also, an `index.html` file needs to be at the root of your repository with your
...
-Once we've packaged our application and did a `Hello World!` in our `index.html`, we must make it available for installation. To achieve this, we can create a [Github](https://github.com/) repository, and push our work on it.
+Once we've packaged our application and did a `Hello World!` in our `index.html`, we must make it available for installation.
+
+Thanks to our awesome team members, you've got not only one nor two, but three solutions to deploy your application in your local environment. Let's see what you can use!
+
+#### 1. Serve local files throught a proxy inside the VM
+
+This solution is dedicated to client-side only apps, and is the prefered way to test your app.
+
+In your terminal, simply go to your app directory, and run the `cozy-dev proxify` command.
```
Comment (misstick) [inline source/clientsideapp.md:116]: " ...and run: <strike>the `cozy-dev proxify` command</strike>"
Smaller is better, don't you think?

File: source/clientsideapp.md [hunk 3]
```diff
@@ -105,7 +105,69 @@ Also, an `index.html` file needs to be at the root of your repository with your
...
+
+
+```
+cd myapp
+cozy-dev proxify http://localhost:9104
+```
+
+
+This command uses up to two params that can let you customize the proxy:
```
Comment (misstick) [inline source/clientsideapp.md]: " This command uses up to two params: `URL` and `subdir`. <strike>that can let you customize the proxy:</strike>"

File: source/clientsideapp.md [hunk 4]
```diff
@@ -105,7 +105,69 @@ Also, an `index.html` file needs to be at the root of your repository with your
...
+
+1. a URL: we here use the local VM (instanciated with `cozy-dev vm:start`), but you can safely use any cozy available remotely. _No more need to use a local VM for development_, you can safely use your own Cozy (like `tbl.cozycloud.cc`)
```
Comment (misstick) [inline source/clientsideapp.md]: Why not using title instead? 
Adding line breaks before your note: "_No more need to use a local ..." ?

File: source/clientsideapp.md (hunk 2 above)
Comment (m4dz) [reply source/clientsideapp.md:116]: Hmm, yep, but the layout present code side-by-side with text, so without it, the text part becomes a non-sens…

File: source/clientsideapp.md (hunk 4 above)
Comment (m4dz) [reply source/clientsideapp.md]: > Why not using title instead?

coz' they're details, not subsections?

> Adding line breaks before your note: "_No more need to use a local ..." ?

In fact, this isn't a note, juste an empahasis on some part of the text :)

**Pull request merged**